tea-config-generator/serverless.yml

```diff
@@ -61,6 +74,7 @@ plugins:
   - serverless-offline
   - serverless-s3-local
 custom:
+  loadBalancer: ${env:AWS_TEA_CONFIG_LOADBALANCER, 'arn:aws:elasticloadbalancing:us-east-1:832706493240:listener/app/cmr-services-sit/9bae549c0224bffd/e2610cc0267bd95e'}
   env: ${env:AWS_TEA_CONFIG_ENV, 'sit'}
```

Comment:
This should be pass in via the `--stage` serverless option.

Reply:
I removed the value